---
sensor_msgs/Image color
sensor_msgs/Image depth
sensor_msgs/Image ir
bool success